src/contents/articles/hello-world/index.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Hello World
+title: Introducing Namespace
 author: Marcus Sanatan
 date: 2015-05-24
 template: article.jade
 ---
 
-### This is **namespace**
```

Comment:
Historical relevance of 'Hello World' as before. I aimed for best of both world, using the reference and making an announcement of the group. I'm not against removing this from the article